docs/plugins/mapper-attachments.asciidoc

```diff
@@ -1,6 +1,9 @@
 [[mapper-attachments]]
 === Mapper Attachments Plugin
 
+deprecated[3.0.0, Mapper attachments has been deprecated by <<ingest-attachment>>]
+
```

Comment:
deprecated[3.0.0,The `mapper-attachments` plugin will be replaced by the `ingest-attachment`  plugin in 3.0]